index.html

```diff
@@ -44,19 +54,19 @@ <h1>Lizansh Mathur</h1>
              </p>
                 <br>
                 
-                <div class="button">
                     <button class="rectangle">Start collaboration</button>
-                </div>
+            
             </div>
-               <img class="proLogo" src="./img/image_geometry_2.png" alt="Logo">
+            <img src="./img/image-geometry_1-1.png" class="banner"> 
+            <img src="./img/image_geometry_2.png" class="proLogo">
     </section>
         <br>
     <section id="work">
-        <div class="work-list">
-        <header class="work-header">
-            <h1 class="pro">Projects</h1>
-        </header>
-        <ul>       
+        
+        <ul class="work-list">     
+            <header class="work-header">
+                <h1 class="pro">Projects</h1>
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/LizanshMathur/Portfolio/pull/13#discussion_r1186453222), please consider using `h2`...`h6` tag here, preferably `h2` for semantic reasons and also because using only one `h1` creates hierarchy and gives your page a more logical structure. ✨📌